client/web/src/hooks/node-data.ts

```diff
@@ -147,12 +141,12 @@ export default function useNodeData() {
         // then make the prefs PATCH. If the request fails,
         // data will be updated to it's previous value in
         // onComplete below.
-        setData(optimisticUpdates)
+        mutate(optimisticUpdates, false)
```

Comment:
Just for my own understanding: is the `false` here is marking the [revalidate param](https://swr.vercel.app/docs/mutation#parameters) as `false`, correct?

Reply:
Yep! A mutation to change the existing data on the page without needing to revalidate in the background. And the mutation in the `.then` actually does the revalidation.